values.html

```diff
@@ -0,0 +1,189 @@
+<!DOCTYPE html>
+<html lang="en">
+	<!-- This file is a placeholder for a page. Rename this file to the name of the page you are working on. -->
+
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<title>If These Lands Could Talk - Our Values</title>
+		<link rel="preconnect" href="https://fonts.googleapis.com" />
+		<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+		<link
+			href="https://fonts.googleapis.com/css2?family=Josefin+Sans:wght@400;500;700&display=swap"
+			rel="stylesheet"
+		/>
+		<link rel="stylesheet" href="css/normalize.css" />
+		<link rel="stylesheet" href="css/global.css" />
+		<!-- In a new branch, rename this page.css file to the name of that actual page (Example: home.css) -->
+		<link rel="stylesheet" href="css/values.css" />
+	</head>
+
+	<body>
+		<!-- Header section -->
```

Comment:
Please add header navigation (it's in index.html)

Reply:
header / footer add in